src/display/mod.rs

```diff
@@ -97,7 +101,11 @@ impl Dot for WithContext<&Expr, DataType> {
         Command::new("open")
             .arg(format!("/tmp/{name}.html"))
             .output()
-            .expect("Error: this works on MacOS");
+            .or(
+                Command::new("xdg-open")
+                .arg(format!("http://localhost:8000/{name}.html"))
```

Comment:
Why did you do that, please remove everywhere.

Reply:
I have reverted it and created the issue [here](https://github.com/Qrlew/qrlew/issues/229)